packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -253,19 +253,8 @@
 						<Accordion multiple :active-index="[0, 1, 2]">
 							<AccordionTab header="Success criteria">
 								<ul>
-									<li v-for="(constraint, i) in node.state.constraintGroups" :key="i">
-										<h5>{{ constraint.name }}</h5>
-										<vega-chart
-											v-if="riskResults[knobs.postForecastRunId]"
-											:visualization-spec="
-												createOptimizeChart(
-													riskResults[knobs.postForecastRunId],
-													constraint.targetVariable,
-													constraint.threshold,
-													constraint.isMinimized
-												)
-											"
-										/>
+									<li v-for="(_constraint, key) in node.state.constraintGroups" :key="`success_criteria_${key}`">
```

Comment:
Whats the point of the :key being a static string concat with the index?

Reply:
I thought it was screwing with other list objects with the same key but thats not even an issue 😆. Its not needed, Ill make it just `key`